examples/stackbreakdown.ipynb

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
can you make this notebook a bit more step-by-step with explanatory text and show how to plot and interact with the results?